flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/util/PlannerConfigImplUtils.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.util;
+
+import org.apache.flink.api.common.time.Time;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.table.api.PlannerConfig;
+import org.apache.flink.table.api.PlannerConfigImpl;
+import org.apache.flink.table.api.TableConfig;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import scala.concurrent.duration.Duration;
+
+import static org.apache.flink.table.api.ExecutionConfigOptions.SQL_EXEC_DISABLED_OPERATORS;
+import static org.apache.flink.table.api.ExecutionConfigOptions.SQL_EXEC_EMIT_EARLY_FIRE_DELAY;
+import static org.apache.flink.table.api.ExecutionConfigOptions.SQL_EXEC_EMIT_EARLY_FIRE_ENABLED;
+import static org.apache.flink.table.api.ExecutionConfigOptions.SQL_EXEC_EMIT_LATE_FIRE_DELAY;
+import static org.apache.flink.table.api.ExecutionConfigOptions.SQL_EXEC_EMIT_LATE_FIRE_ENABLED;
+import static org.apache.flink.table.api.OptimizerConfigOptions.SQL_OPTIMIZER_AGG_PHASE_STRATEGY;
+
+/**
+ * Utility class for {@link PlannerConfigImpl} related helper functions.
+ */
+public class PlannerConfigImplUtils {
```

Comment:
I think this utility is unnecessary, some interfaces are only used by tests, and some of them can directly provided by `PlannerConfigImpl` itself.